dev/benchmarks/platform_views_layout/test_driver/scroll_perf_ad_banners_test.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
+
+void main() {
+  group('scrolling performance test', () {
+    late FlutterDriver driver;
+
+    setUpAll(() async {
+      driver = await FlutterDriver.connect();
+
+      await driver.waitUntilFirstFrameRasterized();
```

Comment:
Try removing this line, I wonder if its interacting poorly with CI.

Reply:
Ah interesting. I simply copied this file. Let me try if this is the issue.

Reply:
Circling back - Tried removing it and doesn't fix the CI failure. I think I will leave it as it is